packages/frontend/src/components/Button/style.module.scss

```diff
@@ -1,9 +1,9 @@
 .button {
   align-items: center;
-  background-color: transparent;
+  background-color: var(--buttonSecondaryBackground);
   border-radius: 10px;
-  border: 0;
-  color: var(--buttonText);
+  color: var(--buttonSecondaryText);
+  border: 1px solid #ccc;
```

Comment:
Three-dot button in "create new profile" has a border, which seems unintentional

![image](https://github.com/user-attachments/assets/518d283c-70a9-463a-9591-ffa1b656e123)

Reply:
is fixed